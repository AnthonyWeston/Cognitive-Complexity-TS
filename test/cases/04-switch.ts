const s: string = "string";
switch (s) {
    case "strong":
        console.log();
    case "strung":
        console.log();
    case "string":
    default:
        console.log();
}