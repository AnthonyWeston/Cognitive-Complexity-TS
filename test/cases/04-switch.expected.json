{
    "04-switch.ts": {
        "score": 1
    }
}